cds/modules/opencast/tasks.py

```diff
@@ -328,7 +328,7 @@ def on_transcoding_completed(
     deposit_id = flow_task.payload["deposit_id"]
     try:
         deposit_video = deposit_video_resolver(deposit_id)
-    except PIDDeletedError:
+    except (PIDDeletedError, PIDDoesNotExistError):
```

Comment:
I am thinking now that the query below `FlowTaskMetadata.query.get(flow_task_id)` might return `None` if the record was deleted (and as consequence, flow and tasks deleted).
Maybe it is safer to add a condition?

Reply:
Hmm, I understand that if it is a soft delete(`PIDDeletedError`) then we get the `flow_task` and update it, but if we hard delete(PIDDoesNotExistError) then we can just exit since there is nothing to process.